packages/admin/admin/src/error/errordialog/createErrorDialogApolloLink.tsx

```diff
@@ -20,9 +20,9 @@ export const createErrorDialogApolloLink = (config?: { signInUrl?: string }) =>
         let httpStatus: string | undefined;
 
         if (graphQLErrors) {
-            if (graphQLErrors.some((e) => e.message === "UNAUTHENTICATED")) {
+            if (graphQLErrors.some((e) => e.extensions?.code === "FORBIDDEN")) {
                 errorType = "unauthenticated"; // Error is triggered by Comet Guard
-            } else if (graphQLErrors.some((e) => e.extensions?.response?.statusCode === StatusCodes.UNAUTHORIZED)) {
+            } else if (graphQLErrors.some((e) => e.extensions?.code === "UNAUTHENTICATED")) {
```

Comment:
@fraxachun I've updated this to work with the new graphql version. However, it seems to me that this has been broken since upgrading to graphql v16 in the API. Also, it's strange that the Comet Guard results in "FORBIDDEN" whereas an UnauthorizedException results in "UNAUTHENTICATED". 🤔